app/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import Alert from '@/components/AlertNotification/AlertNotification';
+import { AlertVariants } from '@/components/AlertNotification/Alerts.enum';
+import { toast } from 'react-hot-toast';
+import { onWeeklyPickChange } from './WeekHelper';
+import { createWeeklyPicks } from '../../../../../../api/apiFunctions';
+import { parseUserPick } from '../../../../../../utils/utils';
+
+jest.mock('../../../../../../api/apiFunctions', () => ({
+  createWeeklyPicks: jest.fn(),
+}));
+
+jest.mock('react-hot-toast', () => ({
+  toast: {
+    custom: jest.fn(),
+  },
+}));
+
+describe('Week', () => {
+  const data = {
+    target: { value: 'Browns' },
+    preventDefault: jest.fn(),
+    stopPropagation: jest.fn(),
+  } as any;
+  const NFLTeams = [{ teamName: 'Browns', teamId: 'mockId' }];
```

Comment:
For the ID, use a string number, just like the DB. It doens't have to be long, but it should be numbers.